draft-ietf-cats-framework.md

```diff
@@ -3,7 +3,7 @@ title: A Framework for Computing-Aware Traffic Steering (CATS)
 abbrev: CATS Framework
 category: info
 
-docname: draft-ietf-cats-framework-latest
+docname: draft-ietf-cats-framework-05
```

Comment:
need to modify to latest

Reply:
No, this is set automatically. Worse, changing this will break CI/CD of the repo

Reply:
ok